astroid/brain/brain_typing.py

```diff
@@ -106,23 +109,32 @@ def __class_getitem__(cls, item):
 """
 
 
-def looks_like_typing_typevar_or_newtype(node):
+def looks_like_typing_typevar(node):
```

Comment:
```suggestion
def looks_like_typing_typevar(node: nodes.Call) -> bool:
    """Checks whether a Call node is a TypeVar call"""
```

Could you add an `from astroid import nodes` line? I can't suggest it because it is outside of the original scope of the commits.